### DISCUSSION TITLE
"Activate" piece of html code in Layout.cshtml when module enabled

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#9669] [Q&A] [answered]
how can i "Activate" piece of html code in Layout.cshtml when module enabled.
Is there any example?

**Comment by jptissot** [comment, marked as answer]
Something like this: https://github.com/StatCan/StatCan.OrchardCore/blob/master/src/Themes/DigitalAdmin/Views/Layout.cshtml#L64 ?

**Reply by vitalybrandes** [reply]
Thank you!
